tests/conftest.py

```
@@ -0,0 +1,48 @@
+
+
+@pytest.fixture
+def runner():
+    """Create a Click CLI test runner."""
+    return CliRunner()
+
+
+@pytest.fixture
+def mock_gh_command(mocker):
+    """Mock the subprocess calls to gh CLI with realistic return values."""
+    from unittest.mock import MagicMock
```

Pylint (3.3.7)

[convention] Import outside toplevel (unittest.mock.MagicMock)

(C0415)